### DISCUSSION TITLE
Value on custom validator null

### DISCUSSION THREAD

**Discussion by rodolphonetto** [adonisjs/core#1377] [Help]
Hello guys, I'm trying to get a value to validate if I can delete something or not. But value is null, because it is a param of router for example:    localhost:3333/deleteunity/1

I need that 1

If I pass localhost:3333/deleteunity/1?data=1

Then I have it

**Comment by iammrsea** [comment]
How did you define your endpoint for that route? Something like **/deleteunity/:id**? And how have you been accessing the parameter?

**Reply by rodolphonetto** [reply]
Yes, like that and I access in the controller like const { id } = params

**Reply by iammrsea** [reply]
Try logging params to the console and see what you have

**Comment by fnoquiq** [comment]
```ts
localhost:3333/deleteunity/2?data=1

const { data } = request.get() // data = 1

params.id // 2

```
